projects.html

```diff
@@ -71,20 +71,18 @@ <h1>PROJECTS</h1>
                         <h1>Overview of Rovers</h1>
 
                         <p>
-                            The NASA Robotics Mining Competition is a
-                            competition hosted by NASA, where fifty teams across
-                            the United States compete to build the best rover
-                            that can mine under a simulated surface of the moon.
-                            A rover starts at the home base, navigates through
-                            obstacles, and then reaches a mining zone where it
-                            has to break through a few layers. It then digs out
-                            regolith (gravel) and transports it back to the home
-                            base, where it deposits the regolith. The process is
-                            repeated multiple times.
+                            <br/>
```

Comment:
This paragraph is confusing. It makes me think that both the RMC and URC only have 50 teams, which is not the case. The paragraph also continues to explain the rules of the NASA RMC, which I think is not needed. 

Perhaps something like this? You might need to word smith this. 
```
TrickFire Robotics has built rovers for the NASA Robotics Mining Competition (RMC) and the University Rover Challenge (URC). The NASA RMC pitted fifty teams across the United States to construct a rover that is capable of mining through a simulated lunar surface. The URC has over a hundred teams, both domestic and international, where a built rover has to be able to manipulate different objects, detect life, and operate autonomously. 

In seven years, TrickFire has built four different rovers, with a fifth one being worked on. 

```